sys/txs-lib/license.txt

```diff
@@ -0,0 +1,29 @@
+BSD 3-Clause License
+
+Copyright (c) 2017, TorXakis
```

Comment:
Do we need to update to 2018?
Or change license to something like  2017 - 2018 
or just 2017 -

Reply:
Do we need this licence file at all? We already have ours in the [root](https://github.com/TorXakis/TorXakis/blob/feat/txs-interface/LICENSE). It also says 2017 though...